nulluniqueidentifier.go

```diff
@@ -0,0 +1,39 @@
+package mssql
+
+// NullUniqueIdentifier represents a GUID that may be null.
+// NullUniqueIdentifier implements the Scanner interface so it can be used as a scan destination.
+type NullUniqueIdentifier struct {
+	UniqueIdentifier UniqueIdentifier
+	Valid            bool
+}
+
+// Scan implements the Scanner interface.
+func (nui *NullUniqueIdentifier) Scan(v interface{}) error {
+
+	if v == nil {
+
+		nui.Valid = false
+		return nil
+	}
+
+	err := nui.UniqueIdentifier.Scan(v)
+	if err != nil {
```

Comment:
Converting any possible scan error into a Null value doesn't seem right to me, error checking should be more specific